lxd/api_cluster.go

```diff
@@ -3736,3 +3742,14 @@ func clusterGroupValidateName(name string) error {
 
 	return nil
 }
+
+// internalClusterNetworkRestart is used to trigger a restart of all networks on the target member.
+func internalClusterNetworkRestart(d *Daemon, r *http.Request) response.Response {
```

Comment:
Thinking more about this, and taking into account the changes of `networkStartup` in https://github.com/lxc/lxd/pull/10056 (namely that it will start a background go routine to keep trying failed networks) is there a reason we cannot update this route to take a specific network name (or names) and just try to start them instead, rather than restarting all networks?